src/graph/InstallPluginExecutor.cpp

```diff
@@ -0,0 +1,61 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include "base/Base.h"
+#include "graph/InstallPluginExecutor.h"
+#include <dlfcn.h>
+
+namespace nebula {
+namespace graph {
+
+InstallPluginExecutor::InstallPluginExecutor(Sentence *sentence,
+                                             ExecutionContext *ectx)
+    : Executor(ectx, "install_plugin") {
+    sentence_ = static_cast<InstallPluginSentence*>(sentence);
+}
+
+Status InstallPluginExecutor::prepare() {
+    return Status::OK();
+}
+
+void InstallPluginExecutor::execute() {
+    pluginName_ = sentence_->pluginName();
+    soName_ = sentence_->soName();
+    auto future = ectx()->getMetaClient()->installPlugin(*pluginName_, *soName_);
+    auto *runner = ectx()->rctx()->runner();
+
+    auto cb = [this] (auto &&resp) {
+        if (!resp.ok()) {
+            doError(Status::Error("Install plugin %s soname \"%s\" failed: %s",
```

Comment:
`doError` and `doFinish` will judge if it needs to do stat, so here doesn't need.

Reply:
Good catch. I got.